ONSdigital/response-operations-ui
# Frontend quick wins

# Motivation and Context
Whilst we are in a slight lull, this is some minor frontend refactors that improve a few things here and there.

# What has changed
* Minor change to watch task
* CSS refactoring as appropriate
* Deletion of old CSS files

# How to test?
* run solution and dependencies
* run acceptance tests
* look at resops to ensure it looks as it should
